Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs

```diff
@@ -82,14 +85,19 @@ void OnGUI()
 		else
```

Comment:
Heartbreaking that Mono isn't up to .NET 4.0, because that `String.isNullOrWhitspace` is a pretty fly call.